packages/forms/src/UIForm/fieldsets/Array/DefaultArrayTemplate.component.js

```diff
@@ -34,7 +35,7 @@ function DefaultArrayTemplate(props) {
 				className={classNames(theme['tf-array-add'], 'tf-array-add')}
 				bsStyle={'info'}
 				onClick={onAdd}
-				label={t('ARRAY_ADD_ELEMENT', { defaultValue: 'New Element' })}
+				label={options.btnLabel || t('ARRAY_ADD_ELEMENT', { defaultValue: 'New Element' })}
```

Comment:
this is the only options you wanna pass, so why an object, why not just addBtnLabel as a string directly in the uiSchema ?

Reply:
@jmfrancois so, initially I've done like this, but according to @jsomsanith comments there is a convention to not put custom properties directly in uiSchema

please read @jsomsanith comments above